lib/livebook/session/data.ex

```diff
@@ -549,7 +551,10 @@ defmodule Livebook.Session.Data do
     |> update_section_info!(section.id, fn section ->
       %{section | evaluation_queue: section.evaluation_queue ++ [cell.id]}
     end)
-    |> set_cell_info!(cell.id, evaluation_status: :queued)
+    |> set_cell_info!(cell.id,
+      evaluation_status: :queued,
+      evaluation_start_time: System.monotonic_time()
```

Comment:
I think it would be more important to start counting from when evaluation starts. To be more precise, we want to know how long the code takes, not necessarily how long it took for us to get the result (which may include other factors). For this reason, it may be more precise to measure the time taken in the evaluator itself and not in Livebook. Does that make sense?

Reply:
Definitely makes sense. We can extend the `Runtime` protocol to require evaluation time in the response message. The `Evaluator` would then just measure the exact time and it to us.

To show the ticking time on the client we will still need a rough estimate of the starting point, but that doesn't have to be precise, so on the client we can detect when the cell goes into evaluating state and handle everything there.

WDYT?

Reply:
I added the time to the evaluator, this is what it did initially. For the client, `data.ex` can keep the `start_time_ms` to start the clock.

Maybe display the precise time in the end

Reply:
Yeah, so I'm thinking we don't even need `start_time_ms`, we would just record the time on the client (JS) as the cell gets in to evaluating state. To show the final time, we measure it in Evaluator, send as part of response, and then rendering for the evaluated status.